### PR TITLE
Remove unused config stuff

### PR DESCRIPTION
**Pull request by bfish713**
Closes #<ISSUE_NUMBER>

### CODE REVIEW

File: crates/hotshot/src/traits/networking/libp2p_network.rs [hunk 1]
```diff
@@ -664,11 +636,6 @@ impl<K: SignatureKey + 'static> Libp2pNetwork<K> {
                 // Subscribe to the QC topic
                 handle.subscribe(QC_TOPIC.to_string()).await.unwrap();
 
-                // Only subscribe to DA events if we are DA
-                if is_da {
-                    handle.subscribe("DA".to_string()).await.unwrap();
-                }
-
```
Comment (ss-es) [inline crates/hotshot/src/traits/networking/libp2p_network.rs:671]: Is the intention that everyone is always receiving all DA broadcasts/gossip?

**Review by ss-es — APPROVED** [review]
looks good to me, just one question

**Review by ss-es — CHANGES_REQUESTED** [review]
oh, wait -- it looks like the PR broke `tests_3` somehow

feel free to dismiss

**Comment by bfish713** [comment]
Figured out test 3, basically the byzantine test for bad proposals is not great in general.  The network task effectively blocks us from sending the multiple bad proposals out.  The reason this PR broke it is because I removed round start delay which changed the timing such that the byzantine node blocks it's own proposal because it thinks it's a duplicate action.  I fixed this by adding more nodes so that we have enough honest leaders to complete 50 other views in time.  I feel like the byzantine leader timing out is totally a reasonable expectation but lmk if you think otherwise

**Review by ss-es — APPROVED** [review]

**Pull request merged**